partner_event/hooks.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 David Vidal
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    """Preload proper attendee partner for existing registrations using
+       the same rules the module does"""
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        attendees = env['event.registration'].search([
+            ('email', '!=', False),
+        ])
+        for attendee in attendees:
+            attendee_partner = Partner.search([
+                ('email', '=ilike', email)
+            ], limit=1)
+            if attendee_partner:
+                attendee.attendee_partner_id = attendee_partner
+            elif attendee.partner_id:
+                attendee.attendee_partner_id = attendee.partner_id
```

Comment:
Not sure of this part. Can this be False?

Reply:
Well, the only required field for an attendee is the event_id

Reply:
Then better to leave it unfilled

Reply:
ok then